### PR TITLE
formatted xml files

### PR DESCRIPTION
**Pull request by akshitdewan**
removed help file from pr altogether, as it always seems to download with very different code

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/WebHome.xml [hunk 1]
```diff
@@ -20,24 +19,246 @@
...
+  <contentUpdateDate>1512001811000</contentUpdateDate>
   <version>1.1</version>
   <title/>
   <comment/>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
-  <hidden>true</hidden>
-  <content/>
+  <hidden>false</hidden>
+  <content>{{include reference="Help.Code.VelocityMacros" /}}
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/WebHome.xml]: This does not seems to be the same content att all. Looks like you exported an older version or something like this.

Comment (akshitdewan) [reply xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/WebHome.xml]: Okay I’ll take a look

**Comment by tmortagne** [comment]
Looks OK this time. Last request: make sure to execute `mvn xar:format` twice since there is a bug that makes it miss an empty line the first time it's executed.

**Comment by akshitdewan** [comment]
Okay will do!

**Comment by akshitdewan** [comment]
@tmortagne I reran mvn format many times, it is not removing the extra line in flamingothemescode, but it did remove it in others

**Comment by tmortagne** [comment]
What I said is that xar:format is supposed to ADD an empty line so it seems to be you executed xar:format only once on other projects.

**Comment by akshitdewan** [comment]
@tmortagne I think its good now

**Comment by tmortagne** [comment]
All good now. Thanks for your contribution !

**Comment by Enygma2002** [comment]
@tmortagne Again, date modifications should not have been committed.

File: xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemes/WebHome.xml [hunk 2]
```diff
@@ -20,19 +20,19 @@
...
-<xwikidoc version="1.2" reference="IconThemes.WebHome" locale="">
+<xwikidoc version="1.3" reference="IconThemes.WebHome" locale="">
   <web>IconThemes</web>
   <name>WebHome</name>
   <language/>
   <defaultLanguage/>
   <translation>0</translation>
   <creator>xwiki:XWiki.Admin</creator>
-  <creationDate>1440512189000</creationDate>
+  <creationDate>1440533789000</creationDate>
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemes/WebHome.xml]: For the record, these date changes should have been excluded from the commit.

File: xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemes/WebHome.xml [hunk 3]
```diff
@@ -20,19 +20,19 @@
...
   <parent>Main.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1440512189000</date>
-  <contentUpdateDate>1440512189000</contentUpdateDate>
+  <date>1512001721000</date>
+  <contentUpdateDate>1512001721000</contentUpdateDate>
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemes/WebHome.xml]: These as well. Same for the other files.

**Pull request merged**